src/main/java/wooteco/subway/admin/domain/Line.java

```diff
@@ -78,20 +94,87 @@ public void update(Line line) {
         if (line.getIntervalTime() != 0) {
             this.intervalTime = line.getIntervalTime();
         }
+        if (line.getColor() != null) {
+            this.color = line.getColor();
+        }
+        if (line.getLineStations() != null) {
+            this.lineStations = line.getLineStations();
+        }
 
         this.updatedAt = LocalDateTime.now();
     }
 
     public void addLineStation(LineStation lineStation) {
-        // TODO: 구현
+        if (lineStations.isEmpty()) {
+            validateFirstLineStation(lineStation);
+        }
+        lineStations.add(lineStation);
+
+        Long preStationId = lineStation.getPreStationId();
+        Long stationId = lineStation.getStationId();
+
+        Iterator iterator = lineStations.iterator();
+        while (iterator.hasNext()) {
+            LineStation selectedLineStation = (LineStation)iterator.next();
+            if (selectedLineStation.getPreStationId().equals(preStationId)
+                && !selectedLineStation.getStationId().equals(stationId)) {
```

Comment:
```suggestion
            if (selectedLineStation.isMatchPreStationId(preStationId)
                && !selectedLineStation.isMatchStationId(stationId)) {
```

getter 대신 메시지를 던지도록 구현해보면 어떨까요?

Reply:
해당 로직은 삭제하였으나, preStationId와 stationId가 일치하는지 확인할 때 getter를 사용하지 않고 LineStation 객체에게 메시지를 던지도록 수정했습니다! :)